modules/http/http-curl-compression.c

```diff
@@ -28,7 +28,63 @@ gint8 CURL_COMPRESSION_DEFAULT = CURL_COMPRESSION_UNCOMPRESSED;
 gchar *CURL_COMPRESSION_LITERAL_ALL = "all";
 gchar *curl_compression_types[] = {"identity", "gzip", "deflate", "brotli", "zstd"};
 
-gboolean http_dd_compress_string(GString *compression_destination, const GString *message, const gint compression)
+gchar *_compression_error_message = "Failed due to %s error. Sending uncompressed data.";
+static inline void _handle_compression_error(GString *compression_dest, gchar *error_description)
+{
+}
+static inline gboolean _raise_compression_status(GString *compression_dest, int algorithm_exit)
 {
   return FALSE;
+}
+
+int _gzip_string(GString *compressed, const GString *message);
+int _deflate_string(GString *compressed, const GString *message);
+int _brotli_string(GString *compressed, const GString *message);
+int _zstd_string(GString *compressed, const GString *message);
```

Comment:
[optional]

It would be cleaner to implement these functions here, so these lines can be removed.

Reply:
The OO refactor removed them completely.